### PR TITLE
Update the slack channel to internals

### PR DESCRIPTION
**Pull request by luzizhuo**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.1 (TiDB 6.1 versions)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/8389
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by sunzhaoyang — APPROVED** [review]

**Review by shichun-0415 — APPROVED** [review]

**Review by leoppro — APPROVED** [review]
LGTM

File: dm/dm-overview.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ tiup install dm dmctl
 
 - [GitHub](https://github.com/pingcap/tiflow/tree/master/dm)
 - [AskTUG](https://asktug.com/tags/dm)
-- [Slack](https://slack.tidb.io/invite?team=tidb-community&channel=sig-migrate&ref=github_sig)
+- [TiDB Internals](https://internals.tidb.io/)
```
Comment (shichun-0415) [inline dm/dm-overview.md:70]: @luzizhuo 或许这里也需要删掉，这里大标题是“社区技术支持”，用于帮助用户解决他们“无法解决的问题”。

Comment (luzizhuo) [reply dm/dm-overview.md:70]: > @luzizhuo 或许这里也需要删掉，这里大标题是“社区技术支持”，用于帮助用户解决他们“无法解决的问题”。

是的我修改一下，多谢。

**Pull request merged**